rep-2000.rst

```diff
@@ -823,6 +823,119 @@ Package manager use for dependencies:
+| OpenCV      |     4.5.4      |     4.5.4      |  3.4.6   |     4.2.0     |      4.5.1       |  4.1.0 / 3.2.0****  |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| OpenSSL     |     1.1.1l     |     1.1.1l     |  1.1.1g  |     1.1.1f    |      1.1.1i      | 1.1.1d / 1.1.1b**** |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Python      |     3.10.0     |     3.8.3      |  3.6.8   |     3.8.2     |      3.9.1       |  3.8.2 / 3.7.5****  |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Qt          |     5.15.2     |    5.12.10     |  5.12.5  |     5.12.3    |      5.15.2      | 5.14.1 / 5.12.5**** |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                              |                                   **Linux only**                                   |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| PCL         |     1.12.0     |      N/A       |  1.12.0  |      N/A      |      1.11.1      |       1.10.0        |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+|                                         **RMW DDS Middleware Providers**                                          |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Cyclone DDS |                                          0.8.x (Réplique)                                           |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Fast-DDS    |                                                2.4.x                                                |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+| Connext DDS |              5.3.1              |   N/A    |     5.3.1     |                  N/A                   |
++-------------+----------------+----------------+----------+---------------+------------------+---------------------+
+
+" * " means that this is not the upstream version (available on the official Operating System repositories) but a package distributed by OSRF or the community (package built and distributed on custom repositories).
+
+" ** " Rolling distributions will see multiple version changes of these dependencies during their lifetime.
+The versions shown for OpenEmbedded are those provided by the 3.1 Dunfell release series; the versions provided by the other supported release series are listed here: https://github.com/ros/meta-ros/wiki/Package-Version-Differences .
```

Comment:
I know this was copied from earlier, but it actually seems out of place as part of the `**` comment.  Maybe this should actually be `***`?

Reply:
I've been digging into this a bit and I think it is intended for `**`. I think that it is intended as a conditional statement on the second part of the note with `**`: "rolling dependencies will see multiple versions" and then, if `**` is applied to OpenEmbedded (which it is), then "versions are provided by the 3.1 Dunfell release." 

This statement has been copied down since Dashing. I cannot speak to its correctness. It also seems that `***` is used in Dashing and seems to make sense to me.

I would also be in favor of deleting this, since a condition on a note is confusing.

Reply:
Yeah, agreed.  Let's remove this for now, we can always add something back if the OpenEmbedded folks provide more clarity.

Reply:
Fixed in https://github.com/ros-infrastructure/rep/pull/326/commits/ce8b91a5f82f28e471c3909443ccc32e6a35beb2